bindings/python/test_m68k.py

```diff
@@ -40,12 +40,14 @@
 	16: "Absolute Data Addressing  - Short",
 	17: "Absolute Data Addressing  - Long",
 	18: "Immidate value",
-}    
+}
 
 def print_insn_detail(insn):
     if len(insn.operands) > 0:
         print("\top_count: %u" % (len(insn.operands)))
 
+	print("\tgroups_count: %u" % len(insn.groups))
```

Comment:
@emoon 
This will cause an exception as insn.groups didn't exist

@aquynh, @emoon :
Only public cs_m68k structure is binded to python. (like other arch)
"detail->groups_count" is private.

In other arch, that's the same behavior on x86, you can't access to insn->detail, insn->detail->group in python

May groups field should move to cs_m68k ... an others arch ?